examples/tasks_tutorial.py

```diff
@@ -177,34 +177,35 @@
 # Example
 # #######
 #
-# Let's create a classification task on a new dataset. We first upload the dataset
-# (just a copy of the Iris dataset in this example) and then create a new classification
-# task on it. We'll use 10-fold cross-validation (ID=1), without a predefined measure.
-
-dataset = openml.OpenMLDataset(
-    "TaskCreationTestDataset",
-    "test",
-    data_format="arff",
-    url="https://www.openml.org/data/download/61/dataset_61_iris.arff",
-)
-dataset.publish()
-openml.datasets.status_update(dataset.dataset_id, "active")
-
-tasktypes = openml.tasks.TaskTypeEnum
-my_task = openml.tasks.create_task(task_type_id=tasktypes.SUPERVISED_CLASSIFICATION,
-                                   dataset_id=dataset.dataset_id,
-                                   target_name="class",
-                                   estimation_procedure_id=1)
-my_task.publish()
-
-############################################################################
-# Complete list of task types:
-
-vars(tasktypes)
-
-
-############################################################################
+# Let's create a classification task on a dataset. In this example we will do this on the 
+# Iris dataset (ID=61). We'll use 10-fold cross-validation (ID=1), and predicive accuracy
+# as the predefined measure (as explained above, this can also be left open). 
+# Note that, since it already has this task, this will return an exception, but if 
+# the task is new it will be created.
+
+try:
+    tasktypes = openml.tasks.TaskTypeEnum
+    my_task = openml.tasks.create_task(
+        task_type_id=tasktypes.SUPERVISED_CLASSIFICATION,
+        dataset_id=61,
+        target_name="class",
+        evaluation_measure="predictive_accuracy",
+        estimation_procedure_id=1)
+    my_task.publish()
+except openml.exceptions.OpenMLServerException as e:
+    # Error code for 'task already exists'
+    if e.code == 614:
+        # Lookup task
+        tasks = openml.tasks.list_tasks(data_id=61, output_format='dataframe')
+        tasks = tasks.query("task_type == 'Supervised Classification' " \
+                            "and estimation_procedure == '10-fold Crossvalidation' " \
+                            "and evaluation_measures == 'predictive_accuracy'")
+        task_id = tasks["tid"].values[0]
```

Comment:
Pandas has a new [to_numpy](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_numpy.html) functionality, could you please use that?